### DISCUSSION TITLE
Welcome to React ChatBotify discussions!

### DISCUSSION THREAD

**Discussion by tjtanjin** [tjtanjin/react-chatbotify#7] [Announcements]
## 👋 Hello and welcome!
While it is encouraged to use [Discord](https://discord.gg/6R4DK4G5Zh) as the main platform for connecting with other members of the community, Discussions also serve as an alternative for you to:
  * Ask questions you’re wondering about
  * Share ideas
  * Engage with other community members
  * Have fun with an open-mind

Feel free to kickstart discussions below and share more about what you do!

**Comment by vencronys** [comment]
You are saving us a lot of work after simple-chatbot support got dropped. The docs are epic as well. Thanks man.

**Reply by tjtanjin** [reply]
Thank you, glad you find it useful 😊

**Comment by AKSinsane1997** [comment]
hi @tjtanjin , i want to change the language attribute in voice option like"hi-IN" , but there is no property in the documentation. kindly update on this please

**Reply by tjtanjin** [reply]
> hi @tjtanjin , i want to change the language attribute in voice option like"hi-IN" , but there is no property in the documentation. kindly update on this please

Hi @AKSinsane1997, you can actually find it in the description of the `language` property under the `audio` section: https://react-chatbotify.tjtanjin.com/docs/api/bot_options#audio

**Reply by AKSinsane1997** [reply]
hi @tjtanjin , thanks for the earlier response, no not the audio property i am talking about the **voice** section, and there is no property for voice section. i have also gone through your node module as well but unable to modify the window.SpeechRecognition. could you please help me out???****

**Reply by tjtanjin** [reply]
Hey @AKSinsane1997, I see what you mean. I can explore adding a `language` option for `voice` as well in the next 1-2 days. Feel free to join the [discord](https://discord.gg/6R4DK4G5Zh) if you'd like to keep up to date!
